include/RAJA/internal/exec-cuda/forall_cuda.hxx

```diff
@@ -316,10 +318,11 @@ RAJA_INLINE void forall(IndexSet::ExecPolicy<seq_segit, cuda_exec<BLOCK_SIZE, As
  *
  ******************************************************************************
  */
-template <size_t BLOCK_SIZE, bool Async, typename LOOP_BODY>
+ /*
```

Comment:
It relies on getSegmentInfo which is deprecated.  What (if anything) do we want here instead?